Xamarin.Forms.Core/Label.cs

```diff
@@ -101,10 +101,16 @@ public virtual string UpdateFormsText(string source, TextTransform textTransform
 			});
 
 		public static readonly BindableProperty PaddingProperty = PaddingElement.PaddingProperty;
-		
+
 		public static readonly BindableProperty TextTypeProperty = BindableProperty.Create(nameof(TextType), typeof(TextType), typeof(Label), TextType.Text,
 			propertyChanged: (bindable, oldvalue, newvalue) => ((Label)bindable).InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged));
 
+		public static readonly BindableProperty AutoFitTextProperty = BindableProperty.Create(nameof(AutoFitText), typeof(AutoFitTextMode), typeof(Label), AutoFitTextMode.None);
+
+		public static readonly BindableProperty MinAutoFitSizeProperty = BindableProperty.Create(nameof(MinAutoFitSize), typeof(int), typeof(Label), 9);
```

Comment:
is this a size, or a font size ? if the latter, the name should reflect that

Reply:
Ive changed it to 

`MaxAutoFitFontSize`
`MinAutoFitFontSize`